backend/app/helpers/spree/admin/base_helper.rb

```diff
@@ -155,6 +155,11 @@ def spree_dom_id(record)
         dom_id(record, 'spree')
       end
 
+      def admin_layout(layout = nil)
+        @admin_layout = layout if layout
+        @admin_layout
+      end
+
```

Comment:
I'd rather avoid having a method, which is both a setter and a getter, as it makes it confusing, and is rather unusual. 

As far as I have seen in other parts of the code, the getter is a method, while the setting is just plain `@admin_layout = 'my-layout'`. Alternatively, if that is too 'internals-revealing', you can do `attr_reader`, `attr_writer`, or all together - `attr_accessor` . See the [`def title`](https://github.com/solidusio/solidus/blob/60ee1dffc7c3932cd7652cc2b4566f08a12f74b8/core/lib/spree/core/controller_helpers/common.rb#L18-L33) example from the code.